main.go

```diff
@@ -1,18 +1,18 @@
 package main
```

Comment:
Potential Bug: The program will exit with a fatal error if it fails to get the diff of a PR. Consider logging the error and moving on to the next PR instead of exiting the program.